### ISSUE TITLE
Exception: A Parser can not resolve classes when using `$ref`

### ISSUE THREAD

**Issue by brunetton** [koxudaxi/datamodel-code-generator#264]
**Describe the bug**
I'm trying to convert a schema that use an external reference; but this makes the conversion script crashes.
I'm not sure whether this is an implemented feature, but I thought it was reading README.md 

**To Reproduce**

As an example, let's say that I want to generate a Person model that uses an external reference to `definitions.json`:

- `person.json`:
    ```json
    {
        "$schema": "http://json-schema.org/draft-04/schema#",
        "title": "Person",
        "type": "object",
        "additionalProperties": false,
        "properties": {
            "name": {
                "$ref": "definitions.json#/elegantName"
            }
        },
        "required": [
            "name"
        ]
    }
    ```
- `definitions.json`:
    ```json
    {
        "$schema": "http://json-schema.org/draft-04/schema#",
        "elegantName": {
            "type": "string",
            "minLength": 3
        }
    }
    ```

Used commandline:
```
$ datamodel-codegen --input person.json --output person.py
```

**Expected behavior**
I excepted the script to convert the schema in the same way as if the source file were:

```json
{
    "$schema": "http://json-schema.org/draft-04/schema#",
    "title": "Person",
    "type": "object",
    "additionalProperties": false,
    "properties": {
        "name": {
            "type": "string",
            "minLength": 3
        }
    },
    "required": [
        "name"
    ]
}
```

**Version:**
 - OS: Ubuntu
 - Python version: 3.7.5
 - datamodel-code-generator version: 0.6.6

Is this a bug or a not implemented error ?

Thanks a lot !

**Comment by koxudaxi** [comment]
@brunetton 
I'm sorry, the code didn't cover your case.
I have fixed it and released a new version as `0.6.7`

**Comment by brunetton** [comment]
Thanks very much ! It's perfect :+1:
